src/Module/Xrd.php

```diff
@@ -153,8 +153,9 @@ private static function printSystemJSON(array $owner)
 		System::jsonExit($json, 'application/jrd+json; charset=utf-8');
 	}
 
-	private static function printJSON($alias, $baseURL, $owner, $avatar)
+	private static function printJSON(string $alias, array $owner, array $avatar)
 	{
+		$baseURL = DI::baseUrl()->get();
```

Comment:
Since we're depending on a `BaseModule`, this can be simplified further:
```suggestion
		$baseURL = $this->baseUrl->get();
```